src/main/menu.js

```diff
@@ -94,6 +103,10 @@ fluid.defaults("gpii.app.menuInAppDev", {
                 func: "gpii.app.menu.generateMenuTemplate",
                 args: [
                     "{that}.model.showQSS",
+                    // "{that}.options.showCaptureTool",
```

Comment:
Why did these options get commented out?

Reply:
I went back and forth between whether or not to include the capture tool on the task tray menu for both the regular and dev configurations or only for `npm run dev`. At the moment I'm including them for both since we are building a custom installer for the case study.  However, once you've reviewed everything else and this is ready for master I should  remove them from the production task tray, in which case the dev menu uses these `options` rather than inheriting them from the standard menu.

Reply:
bump

Reply:
removed these commented items.